setup.cfg

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 110
+max-doc-length = 79
+ignore = E133, E226, E228, N802, N803, N806, N812, N815, N816, W503
+exclude = DATA_REPO/
+
+[tool:pytest]
+addopts = --flake8
+flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N815 N816 W503
+# The matplotlib test may not release font files
```

Comment:
...and so? It's not clear what the significance of this comment is.

Reply:
Neither do I. I guess that is in the setup.cfg that I initially copied this from. Will remove.

Reply:
Ah. daf_butler has matplotlib tests and has a pytest-open-files configuration option after that line